packages/matchbox/src/components/EmptyState/Image.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { StyledImage } from './styles';
+
+import { Picture } from '../Picture';
+
+const Image = React.forwardRef(function Image(props, userRef) {
+  const { src } = props;
+
+  return (
+    <StyledImage
+      display={['none', null, 'block']}
+      width="100%"
+      maxWidth="600px"
```

Comment:
This feels a _bit_ small to be, maybe because of my monitor. Maybe we could make this larger based on breakpoint?

Reply:
Or maybe it should just expand to the 50% container width?